netket/models/rbm.py

```diff
@@ -127,3 +127,72 @@ def __call__(self, x):
         im = jnp.sum(im, axis=-1)
 
         return re + 1j * im
+
+
+class RBMSymm(nn.Module):
+    """A restricted boltzman Machine, equivalent to a 2-layer FFNN with a
+    nonlinear activation function in between
+
+    Attributes:
+        dtype: dtype of the weights.
+        activation: The nonlinear activation function
+        alpha: feature density. Number of features equal to alpha * input.shape[-1]
+        use_bias: if True uses a bias in the dense layer (hidden layer bias)
+        use_visible_bias: if True adds a bias to the input
+        kernel_init: initializer function for the weight matrix.
+        bias_init: initializer function for the bias.
+        visible_bias_init: initializer function for the visible_bias.
+    """
+
+    permutations: Callable
```

Comment:
What should be the signature?

Reply:
`Callable[[], Array]` - I can add that, though with the constructors this is pretty internal anyways.